scripts/actions/mobskills/feral_peck.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+--  Feral Peck
+--    Mob Ability: 2429
+--  Description: Deals a set amount of heavy damage (seems like ~90% of target's current HP) to a single target.
+--  Type: Physical
+--  Utsusemi/Blink absorb: Needs retail verification
+--  Range: Melee
+--  Notes: Used only by Zirnitra and Turul
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local currentHP = target:getHP()
+    local damage    = target:getHP()
```

Comment:
2 variables of the same thing. you used the 2nd one only (tho the 1st ones name makes more sense) which is causing CI to complain

Reply:
sorry man, fix incoming